dart/web/ui.html

```diff
@@ -80,7 +80,7 @@
                         <li><a href="#/issues/-/policy/-/-/-/-/True/-/1/25">Managed Policy Issues</a></li>
                         <li><a href="#/issues/-/redshift/-/-/-/-/True/-/1/25">Redshift Issues</a></li>
                         <li class="divider"></li>
-                        <li><a href="#/justified/-/-/-/-/-/-/-/1/25">Justified Issues</a></li>
+                        <li><a href="#/justified/-/-/-/-/-/-/-/-/1/25">Justified Issues</a></li>
```

Comment:
Whoops, good catch!  I think we must have prepped this at the same time that @supertom made his changes for the account type :)